graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -124,14 +123,12 @@ def _get_fields_for_class(schema_graph, graphql_types, field_type_overrides, hid
     schema_element = schema_graph.get_element_by_class_name(cls_name)
     outbound_edges = (
         ('out_{}'.format(out_edge_name),
-         schema_graph.get_element_by_class_name(out_edge_name).properties[
-             EDGE_DESTINATION_PROPERTY_NAME].qualifier)
+         next(iter(schema_graph.get_element_by_class_name(out_edge_name).out_connections)))
```

Comment:
This seems like it could benefit from an assertion that the element we're fetching is indeed unique. Try using the `get_only_element_from_collection` function, and consider moving it out of the `compiler` submodule and into some more generic location.